src/wax_contract.cpp

```diff
@@ -108,18 +108,28 @@ CONTRACT WAX_CONTRACT_NAME: public contract {
                 &sig_val, sizeof(sig_val), random_value, sig_it->exponent, sig_it->modulus),
              "Could not verify signature.");
         
-        jobs_table.modify(job_it, get_self(), [&](auto& rec) {
-            rec.random_value = random_value;
-        });
-
-	checksum256 rv_hash = eosio::sha256(random_value.data(),
-          random_value.size());
+        checksum256 rv_hash = eosio::sha256(random_value.data(), random_value.size());
```

Comment:
**eosio::** is not necessary (there is a _using namespace eosio_ at the top of the file)

Reply:
Ok, it wasn't added as part of this PR, but yes, let's remove it.

Reply:
Yes, I know, it came from other PR.